lib/src/model/inheritable.dart

```diff
@@ -125,7 +125,7 @@ mixin Inheritable on ContainerMember {
 
   Inheritable? get overriddenElement;
 
-  /// True if this [Inheritable] is overriding a superclass.
+  /// Whether this [Inheritable] is overriding a member from a superclass.
```

Comment:
Can we also mention the relationship between `isOverride` and `isInherited` like you mentioned before in GVC? Might be good information for later.

Reply:
Done